staging/src/k8s.io/mcs-api/config/crds/multicluster.k8s.io_serviceexports.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: unapproved
+  name: serviceexports.multicluster.k8s.io
+spec:
+  group: multicluster.k8s.io
+  scope: Namespaced
```

Comment:
Is definiting `kind`, `singular` and `plural` needed?

Reply:
```
names:
    kind: ServiceExport
    singular: serviceexport
    plural: serviceexports
```

Reply:
These are defined at the end of the file, after versions